docs/resources/cloud_gateway_transit_gateway.md

```diff
@@ -35,7 +35,7 @@ resource "konnect_cloud_gateway_transit_gateway" "my_cloudgatewaytransitgateway"
       transit_gateway_id = "...my_transit_gateway_id..."
     }
   }
-  aws_vpc_peering_gateway = {
+  create_aws_vpc_peering_gateway = {
```

Comment:
This seems unexpected 🤔

Reply:
Yes, this is unexpected. Can we add a Speakeasy name override to the schema? It can be done directly in the DCGW spec on the `CreateAwsVpcPeeringGateway` `allOf` by adding an additional entry at the bottom of the list

Reply:
Done and updated (PR on platform API - https://github.com/Kong/platform-api/pull/1187)

Reply:
There are still some changes - although none in the generated docs - I need some help with verifying they don't break anything.
The e2e tests ran fine, but I have not manually done CRUD on the cloud gateways to verify.

Reply:
Provider changes are expected for this work. If the docs haven't changed, it's likely that the public interface hasn't changed either. I can run my local CGW setup and make sure it still works

Reply:
That would be great, thanks 🙏

Reply:
The apply went through fine, and looking at the provider diff the changes all look sensible